lib/tori/backend/s3.rb

```diff
@@ -82,46 +102,32 @@ def head(filename = nil)
       end
 
       def public_url(filename)
-        "#{client.config.endpoint}/#{@bucket}/#{filename}"
+        "#{@client.config.endpoint}/#{@bucket}/#{filename}"
```

Comment:
I think, This fix is that there is no relationship with this PR.
And other same change.

```diff
- client
+ @client
```

Could you split other PR or remove ?